### PR TITLE
Try and fix mob prototypes

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR

dear lort it is hell

adds a whole bunch of base entity prototypes for composition:
- **BaseMob**: All of the comps that are literally required for an entity to be playable.
- **BaseMobDamageable**: Used for mobs that can take damage and have mobstates
- **BaseMobCombat**: Mobs that have combat mode and can punch shit (Rad!)
- **BaseMobAtmosExposed**: Mobs that are affected by atmos
- **BaseMobAtmosStandard**: Mobs that need standard atmospheric conditions to live
- **BaseMobFlammable**: Mobs you can set on fire
- **BaseMobRespirator**: Mobs that can breathe shit
- **BaseMobBloodstream**: Mobs that have a bloodstream and can be injected.

cleans up a bunch of species and simplemob base prototypes.
i left the rest of the cleaning up because this drained my brain.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

- BaseMobOrganic was renamed to BaseMobSpeciesOrganic
- Species mobs no longer inherit from both BaseMobOrganic and the abstract entity for the species. They now solely inherit from the abstract entity.
**Changelog**

no cl no fun

### CODE REVIEW

**Comment by metalgearsloth** [comment]
parent composition my beloved

**Comment by deltanedas** [comment]
no more 800 line of animal copy paste

**Comment by Partmedia** [comment]
Under breaking changes, are there more instructions that need to be added on how to migrate custom species that might be present on forks?

**Comment by EmoGarbage404** [comment]
> Under breaking changes, are there more instructions that need to be added on how to migrate custom species that might be present on forks?

Added the part about species mob inheritance that was missing.

File: Resources/Prototypes/Entities/Mobs/NPCs/simplemob.yml [hunk 1]
```diff
@@ -1,98 +1,22 @@
 - type: entity
   save: false
+  parent:
+  - BaseMob
+  - BaseMobDamageable
+  - BaseMobAtmosExposed
```
Comment (DrSmugleaf) [inline Resources/Prototypes/Entities/Mobs/NPCs/simplemob.yml:6]: Please god make the names for these shorter

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Mobs/NPCs/simplemob.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Prototypes/Entities/Mobs/NPCs/simplemob.yml:6]: you clearly havent seen objective rework :trollface: this is very short

Comment (DrSmugleaf) [reply Resources/Prototypes/Entities/Mobs/NPCs/simplemob.yml]: Fixed in [`1798f23` (#19859)](https://github.com/space-wizards/space-station-14/pull/19859/commits/1798f23fdd477a21adb204479a638e0cb0d53d9d)

**Review by DrSmugleaf — APPROVED** [review]

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**